docs/fsharp/tools/fsharp-interactive/index.md

```diff
@@ -249,6 +249,42 @@ The `#r` and `#load` directives seen previously are only available in F# Interac
 
 When you specify files or paths in F# Interactive, a string literal is expected. Therefore, files and paths must be in quotation marks, and the usual escape characters apply. You can use the `@` character to cause F# Interactive to interpret a string that contains a path as a verbatim string. This causes F# Interactive to ignore any escape characters.
 
+### Extended #help directive
+
+The `#help` directive now supports displaying documentation for specific objects or functions. You can pass the name of the object or function directly (without quotes) to retrieve details.
```

Comment:
```suggestion
The `#help` directive now supports displaying documentation for specific functions. You can pass the name of the object or function directly (without quotes) to retrieve details.
```


(I have tried a few use cases and I can only get it to explain functions, not types/modules/objects)